app/paypal.py

```
@@ -0,0 +1,230 @@
+                sql_connection.ping()
+                cursor = sql_connection.cursor()
+                cursor.execute("SELECT 1 FROM purchases WHERE paypal_transaction_id=?", [transaction['transaction_info']['transaction_id']])
+                user_id = str(transaction['transaction_info']['custom_field']).strip(' ')
+                if cursor.fetchone():
+                    continue
+                if is_uuid_like(user_id):
+                    sql_connection.ping()
+                    cursor = sql_connection.cursor()
+                    cursor.execute("SELECT user_id FROM chat_users.users WHERE user_id=?", [user_id])
+                    found = False
+                    for row in cursor:
```

Ruff (0.12.2)

Loop control variable `row` not used within loop body

Rename unused `row` to `_row`

(B007)